content/en/flux/security/slsa-assessment.md

```diff
@@ -0,0 +1,188 @@
+  generation and distribution.
+- The provenance for the release artifacts published to GitHub releases (binaries, SBOMs, deploy manifests, source code)
+  is generated using the `generator_generic_slsa3` GitHub Workflow provided by
+  the [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator).
+- The provenance for the release artifacts published to GitHub Container Registry and to DockerHub (Flux controllers
+  multi-arch container images) is generated using the `generator_container_slsa3` GitHub Workflow provided by
+  the [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator).
+
+### Build Platform Requirements
+
+#### Provenance generation
+
+| Requirement               | Required at SLSA L3 | Met by Flux |
+|---------------------------|---------------------|-------------|
+| Provenance Exists         | Yes                 | Yes         |
+| Provenance is Authentic   | Yes                 | Yes         |
+| Provenance is Unforgeable | Yes                 | Yes         |
+
+> The build process MUST generate provenance that unambiguously identifies the output package by cryptographic digest
+> and describes how that package was produced.
+
+- The Flux project release workflows make use of the
+  official [SLSA GitHub Generator project](https://github.com/slsa-framework/slsa-github-generator) for provenance
+  generation.
+- The provenance file stores the SHA-256 hashes of the release artifacts (binaries, SBOMs, deploy manifests, source
+  code).
+- The provenance identifies the Flux container images using their digest in SHA-256 format.
+
+> Consumers MUST be able to validate the authenticity of the provenance attestation in order to ensure integrity and
+> define trust.
+
+- The provenance is signed by Sigstore Cosign using the GitHub OIDC identity and the public key to verify the provenance
+  is stored in the public [Rekor transparency log](https://docs.sigstore.dev/rekor/overview/).
+- The release process and the provenance generation runs in isolation on an ephemeral environment provided by
+  GitHub-hosted runners.
+- The provenance of the Flux release artifacts (binaries, container images, SBOMs, deploy manifests) can be verified
+  using the official [SLSA verifier tool](https://github.com/slsa-framework/slsa-verifier).
+
+> Provenance MUST be strongly resistant to forgery by tenants.
+
+- The provenance generation workflows run on ephemeral and isolated virtual machines which are fully managed by GitHub.
+- The provenance signing secrets are ephemeral and are generated through
+  Sigstore's [keyless signing](https://github.com/sigstore/cosign/blob/main/KEYLESS.md) procedure.
+- The [SLSA GitHub generator](https://github.com/slsa-framework/slsa-github-generator) runs on separate virtual machines
+  than the build and release process, so that the Flux build scripts don't have access to the signing secrets.
```

Comment:
```suggestion
- The [SLSA GitHub generator](https://github.com/slsa-framework/slsa-github-generator) for an artifact runs on a separate virtual machine
  than the one for the build and release process, so that the Flux build scripts don't have access to the signing secrets.
```

Reply:
It's machines since we have more than one repo/workflow

Reply:
yeah that's why i added "for an artifact", but i guess we can keep the original terminology

Reply:
It's not a single artifact, we don't spinup a VM for every file in the release process, but for all of them.